client-network/src/network_ops.rs

```diff
@@ -23,5 +43,5 @@ pub trait NetworkOpsClient {
         attributes: TxAttributes,
     ) -> Result<TxAux>;
 
-    // TODO: Add `create_deposit_bonded_stake_transaction()`, `create_unbond_stake_transaction()` and `get_account_details()` functions
+    // TODO: Add `get_account_details()` functions
```

Comment:
Now that we have changed name from `Account` to `StakedState`
```suggestion
    // TODO: Add `get_staked_state()` functions
```